tests/test_security_analysis.py

```diff
@@ -151,28 +151,28 @@ def test_provider_parameters():
 
 
 def test_ac_security_analysis_with_report():
-    reporter = rp.Reporter()
-    report1 = str(reporter)
+    report_node = rp.ReportNode()
+    report1 = str(report_node)
     assert len(report1) > 0
     n = pp.network.create_eurostag_tutorial_example1_network()
     sa = pp.security.create_analysis()
     sa.add_single_element_contingency('NHV1_NHV2_1', 'First contingency')
-    sa_result = sa.run_ac(n, reporter=reporter)
-    report2 = str(reporter)
+    sa_result = sa.run_ac(n, report_node=report_node)
```

Comment:
is it useful to add a test to check that retro-compatibility still works when using Reporter ?

Reply:
done.